tests/test_landfill_sampler.py

```diff
@@ -9,6 +10,9 @@
 @pytest.fixture
 def sample_document():
     return {
+        # The "content" field is provided here as a dict for
+        # convenience, but should be serialized to a JSON
+        # string before passing through to the sampler code.
         "content": {"payload": {"foo": "bar"}},
```

Comment:
```suggestion
        "content": serialize_content({"payload": {"foo": "bar"}}),
```

This could reduce the diff by a few lines.